package io.github.claytonmonson.vendas.exception;

public class PedidoNaoEncontradoException extends RuntimeException {
    public PedidoNaoEncontradoException() {
        super("Pedido n√£o encontrado.");
    }
}
